articles/virtual-network/accelerated-networking-overview.md

```diff
@@ -100,9 +100,18 @@ You can directly query the list of VM SKUs that support Accelerated Networking b
 
 ### Custom VM images
 
-If you use a custom image that supports Accelerated Networking, make sure you have the required drivers to work with Mellanox ConnectX-3, ConnectX-4 Lx, and ConnectX-5 NICs on Azure. Accelerated Networking also requires network configurations that exempt configuration of the virtual functions on the mlx4_en and mlx5_core drivers.
+If you use a custom image that supports Accelerated Networking, make sure you meet the following requirements.
 
-Images with cloud-init version 19.4 or later have networking correctly configured to support Accelerated Networking during provisioning.
+#### Device and driver support
+Any custom image supporting Accelerated Networking must include drivers that enable Single Root I/O Virtualization for the network interface cards (NIC) which are used on Azure platforms. This hardware list includes NVIDIA ConnectX-3, ConnectX-4 Lx, ConnectX-5 and the [Microsoft Azure Network Adapter (MANA)](https://learn.microsoft.com/en-us/azure/virtual-network/accelerated-networking-mana-overview). Download and install the latest versions of these drivers into your custom image.
```

Comment:
I suppose we should just remove: 
Download and install the latest versions of these drivers into your custom image.

Reply:
Removed it, is there anything we'd want to say instead?